src/sentry/data/config/config.yml.default

```diff
@@ -80,3 +80,16 @@ filestore.options:
 #   access_key: 'AKIXXXXXX'
 #   secret_key: 'XXXXXXX'
 #   bucket_name: 's3-bucket-name'
+
+#########
+# Relay #
+#########
+
+# statically configured relays have the following structure
+#  relay:
+#    static_auth:
+#      7a4905dd-0eeb-497c-aa49-fb9949906388:   # the relay id
+#        public_key: "SMSesqan65THCV6M4qs4kBzPai60LzuDn-xNsvYpuP8"
+#        internal: true   # if true the relay is internal and will have access to all project configs
```

Comment:
The format is different from what is being used in Relay (https://github.com/getsentry/relay/pull/991). In Relay, we use a flat list, while in Sentry we use a dictionary mapped by Relay-Id. Can we please use the same format on both sides?

It looks like the mapping by ID makes more sense based on how this is being used.